src/course/en/components.json

```
@@ -210,7 +223,7 @@
         "instruction": "Select the play button to start the video.",
         "_media": {
             "mp4": "course/en/video/c-40.mp4",
-            "poster": "course/en/video/c-40.jpg",
+            "poster": "course/en/video/media.png",

```

could you change this back to c-40.jpg?
```suggestion
            "poster": "course/en/video/c-40.jpg",
```